Assets/Plugins/LeapMotion/Internal/VRVisualizer/ScenesResources/Scripts/VisualizerManager.cs

```diff
@@ -91,26 +106,31 @@ void Update()
       }
   
       m_trackingText.text = "Tracking Mode: ";
-      m_trackingText.text += (m_controller.IsPolicySet(Controller.PolicyFlag.POLICY_OPTIMIZE_HMD)) ? "Head-Mounted" : "Desktop";
 
-
-      // In Desktop Mode
-      if (m_PCVisualizer.activeInHierarchy)
+      switch (m_controller.GetTrackingMode())
       {
-        if (m_controller.IsPolicySet(Controller.PolicyFlag.POLICY_OPTIMIZE_HMD))
-        {
-          m_trackingText.text += " (Press '" + keyToToggleHMD + "' to switch to desktop mode)";
-          if (Input.GetKeyDown(keyToToggleHMD))
-            m_controller.ClearPolicy(Controller.PolicyFlag.POLICY_OPTIMIZE_HMD);
-        }
-        else
-        {
-          m_trackingText.text += " (Press '" + keyToToggleHMD + "' to switch to head-mounted mode)";
-          if (Input.GetKeyDown(keyToToggleHMD)) {
-              m_controller.SetPolicy(Controller.PolicyFlag.POLICY_OPTIMIZE_HMD);
-          }
-        }
-      } 
+        case eLeapPolicyFlag.eLeapTrackingMode_Desktop:
+          m_trackingText.text += "Desktop";
```

Comment:
I haven't tried compiling this yet, but this line (and the ones in the other cases) probably shouldn't end with a semi-colon (the next line needs to join on).

Consider using formatted strings: https://docs.microsoft.com/en-us/dotnet/api/system.string.format?view=netcore-3.1.

Reply:
Indeed forgot to fuse the instructions.

Let me know if you are happy now with formating